internal/srctesting/srctesting.go

```diff
@@ -63,3 +65,16 @@ func ParseFuncDecl(t *testing.T, src string) *ast.FuncDecl {
 	}
 	return fdecl
 }
+
+// Format AST node into a string.
+//
+// The node type must be *ast.File, *printer.CommentedNode, []ast.Decl,
+// []ast.Stmt, or assignment-compatible to ast.Expr, ast.Decl, ast.Spec, or
+// ast.Stmt.
+func Format(t *testing.T, fset *token.FileSet, node any) string {
```

Comment:
I would expect a function like this to call `t.Helper()`, but maybe upstream doesn't for some reason?

Reply:
Nope, I just forgot to do it.